templates/main/help.html

```diff
@@ -0,0 +1,459 @@
+{% extends "main/base.html" %}
+{% block title %}Getting Help{% endblock %}
+{% block content %}
+    <div class="uk-section" style="padding-top: 20px; padding-bottom: 30px;">
```

Comment:
another problem that could probably be solved with padding classes and without tag styling